docs/Mercury.md

```diff
@@ -0,0 +1,346 @@
+# Mercury Documentation
+
+## Contracts
+
+Use this tool to configure contracts:
+
+https://github.com/smartcontractkit/the-most-amazing-mercury-contract-configuration-tool
+
+TODO: updated process here @Austin Born 
+
+[Reference contract](https://github.com/smartcontractkit/reference-data-directory/blob/master/ethereum-testnet-goerli-arbitrum-1/contracts/0x535051166466D159da8742167c9CA1eFe9e82613.json)
+
+[OCR2 config documentation](https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2/internal/config/public_config.go)
+
+**🚨 Important config**
+
+`s` - transmission schedule. This should be set to the number of oracles on the feed - meaning that every oracle will attempt to transmit to the mercury server in the first stage of transmission. eg `[4]` if there are 4 node in the DON, excluding the bootstrap node.
+
+`f` - set this to `n//3` (where `//` denotes integer division), e.g. if you have 16 oracles, set `f` to 5.
+
+`deltaRound` - report generation frequency. This determines how frequently a new round should be started at most (if rounds take longer than this due to network latency, there will be fewer rounds per second than this parameter would suggest). `100ms` is a good starting point (10 rounds/s).
+
+`reportingPluginConfig.alphaAccept` - set this to `0`, because our mercury ContractTransmitter doesn't know the latest report that's been sent to the mercury server and we therefore always have a "pending" report which we compare against before accepting a report for transmission.
+
+`reportingPluginConfig.deltaC` - set this to `0` so every round will result in a report.
+
+<details><summary>Example `verifier/<0xaddress>.json`:</summary>
+```json
```

Comment:
For some reason, there must be a new line in between the html tags and the code block for these to render properly :shrug: 
```suggestion
<details><summary>Example `verifier/<0xaddress>.json`:</summary>

```json
```